src/frontend/src/utils/Pipes.ts

```diff
@@ -28,6 +28,11 @@ export const wbsPipe = (wbsNum: WbsNumber) => {
   return `${wbsNum.carNumber}.${wbsNum.projectNumber}.${wbsNum.workPackageNumber}`;
 };
 
+/** Display WBS number as string but always the project number (1.2.3 -> 1.2.0) */
+export const wbsToProjectWbs = (wbsNum: WbsNumber) => {
```

Comment:
Ew but okay

Reply:
Ew cuz you're both destroying the object structure by converting to string and you're switching last digit to 0, so it's a two in one when we already have a function that does one of those two

Reply:
tru